### PR TITLE
Fix typo in Makefile; fix 'make serve' to bind to all interfaces, not just localhost

### PR DESCRIPTION
**Pull request by mikemccand**

### CODE REVIEW

File: engines/tantivy-0.19/src/bin/build_index.rs [hunk 1]
```diff
@@ -62,7 +62,7 @@ fn main_inner(output_dir: &Path, index_delete_pct: i32) -> tantivy::Result<()> {
             // (title, date, body, label)
             let parsed_line: Vec<&str> = line.split('\t').collect();
             if parsed_line.len() != 4 {
-                println!("Skippig malformed line: {}", line);
+                println!("Skipping malformed line: {}", line);
```
Comment (Tony-X) [inline engines/tantivy-0.19/src/bin/build_index.rs:65]: great catch!

File: Makefile [hunk 2]
```diff
@@ -1,5 +1,6 @@
 DEV_CORPUS := $(shell pwd)/corpus/20k-enwiki-20120502-lines-1k-fixed-utf8-with-random-label.txt
-REAL_CORPUS := "$(shell pwd)/corpus/enwiki-20120502-lines-1k-fixed-utf8-with-random-label.txt"
+#REAL_CORPUS := "$(shell pwd)/corpus/enwiki-20120502-lines-1k-fixed-utf8-with-random-label.txt"
+REAL_CORPUS := "/lucenedata/enwiki/enwiki-20120502-lines-1k-fixed-utf8-with-random-label.txt"
```
Comment (Tony-X) [inline Makefile:3]: I'm sure this served you well in your local setup. 

Not sure if we want to check this it in

**Review by Tony-X — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (mikemccand) [reply Makefile:3]: Woops, right, I didn't mean to -- I've now fixed my local area (used symlink) so I don't need that mod locally.

**Comment by mikemccand** [comment]
On second thought -- let's not change to bind to all interfaces -- someone, somewhere will consider this a security hole or so.  I can just ssh tunnel through to the server myself to workaround.

**Pull request closed without merging**